syntax = "proto3";

package archivo;

message ArchivoVacio {}  // Mensaje vac√≠o personalizad

message ArchivoRequest {
  string archivo_buscado = 1;
}

message ArchivoLista {
  repeated string archivos = 1;
}

service Archivo {
  rpc ListarArchivos (ArchivoVacio) returns (ArchivoLista);
  rpc BuscarArchivos (ArchivoRequest) returns (ArchivoLista);
}